DuckDB.NET/DuckDBNativeObjects.cs

```diff
@@ -51,52 +51,21 @@ public enum DuckDBType
         DuckdbTypeBlob
     }
 
-    [StructLayout(LayoutKind.Sequential)]
-    public struct DuckDBColumn
-    {
-        IntPtr data;
-        IntPtr nullmask;
-
-        public DuckDBType Type { get; }
-        public string Name { get; }
-        IntPtr internal_data;
-
-        public bool NullMask(int row) => Marshal.ReadByte(nullmask + row) != 0;
-        public IntPtr Data => data;
-
-        public T ReadAs<T>(int row) where T : struct
-        {
-            return Marshal.PtrToStructure<T>(Data + Marshal.SizeOf<T>() * row);
-        }
-    }
-
     [StructLayout(LayoutKind.Sequential)]
     public class DuckDBResult
     {
-        public long ColumnCount { get; }
-        public long RowCount { get; }
-        public long RowsChanged { get; }
-
+        // Deprecated
+        private long ColumnCount;
```

Comment:
Can you add `Obsolete` to deprecated fields?

Reply:
Should I remove the comment as well?

Reply:
Yes, you can.